Terminal.Gui/Core/View.cs

```diff
@@ -1208,6 +1208,10 @@ public virtual void Redraw (Rect bounds)
 								view.LayoutSubviews ();
 							Application.CurrentView = view;
 
+							if (!view.loaded) {
```

Comment:
This seems like a very weird place to do this. Shouldn't the call to `OnLoaded` come from `Application`? E.g. within `RunLoop`?